ServerSide/yoty/Core/Managers/Suppliers/SuppliersManager.cs

```diff
@@ -57,6 +57,32 @@ public async Task<Response> DeleteSupplier(string supplierId)
 
         }
 
+        public async Task<Response<BidsDTO>> GetBidsWhereChosen(string supplierId, BidsTime timeFilter)
```

Comment:
it will be change such that we will extract supplierId from the token claims

Reply:
cool, make sure to do the same for buyerBids